index.html

```diff
@@ -3456,6 +3456,44 @@ <h2>
         <h2>
 Metadata Structure
         </h2>
+
+        <p>
+Input and output metadata is often involved during the <a>DID Resolution</a>,
+<a>DID URL Dereferencing</a>, and other DID-related processes. The structure
+used to communicate this metadata MUST be a <a data-cite="INFRA#maps">map</a> of
+properties. Each property name MUST be a <a data-cite="INFRA#string">string</a>.
+Each property value SHOULD be a <a data-cite="INFRA#string">string</a>. In the
+rare cases where a non-string value is required, the value and all non-string
+sub-values MUST be expressed as a <a data-cite="INFRA#maps">map</a>, <a
+data-cite="INFRA#lists">list</a>, <a data-cite="INFRA#boolean">boolean</a>,
+number, or  <a data-cite="INFRA#null">null</a> and it MUST be possible to
+losslessly represent the value across all supported representations as
+described in <a href="#core-representations"></a>.
+        </p>
+
+        <p>
+The following example demonstrates a JSON-encoded metadata structure sent to
+the <a href="#did-resolution">DID Resolution process</a>.
+        </p>
```

Comment:
The resolution process should absolutely NOT accept something "JSON-encoded". It should accept a "map". Serialization is internal to the process, on either input or output from the function.

Reply:
Applied your suggestions verbatim.